### PR TITLE
Reformat the open a PR page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/contribute/new-content/open-a-pr.md [hunk 1]
```diff
@@ -157,75 +161,80 @@ Figure 2. Working from a local fork to make your changes.
...
+   git submodule update --init --recursive --depth 1
+   ```
 
-4. Navigate to the new `website` directory. Set the `kubernetes/website` repository as the `upstream` remote:
+1. Navigate to the new `website` directory. Set the `kubernetes/website` repository as the `upstream` remote:
 
-      ```bash
-      cd website
+   ```shell
+   cd website
```
Comment (Sea-n) [inline content/en/docs/contribute/new-content/open-a-pr.md:179]: Do we have any guideline for which ` ```lang ` should we use?

File: content/en/docs/contribute/new-content/open-a-pr.md [hunk 2]
```diff
@@ -157,75 +161,80 @@ Figure 2. Working from a local fork to make your changes.
...
-    Output is similar to:
+   Output is similar to:
 
-    ```bash
-    origin	git@github.com:<github_username>/website.git (fetch)
-    origin	git@github.com:<github_username>/website.git (push)
-    upstream	https://github.com/kubernetes/website.git (fetch)
-    upstream	https://github.com/kubernetes/website.git (push)
-    ```
+   ```none
```
Comment (Sea-n) [inline content/en/docs/contribute/new-content/open-a-pr.md:192]: Should we use empty ` ``` `, ` ```console ` or ` ```none `?

File: content/en/docs/contribute/new-content/open-a-pr.md [hunk 3]
```diff
@@ -235,116 +244,124 @@ When you are ready to submit a pull request, commit your changes.
...
+   or close the terminal window.
 
 {{% /tab %}}
 {{% tab name="Hugo on the command line" %}}
 
 Alternately, install and use the `hugo` command on your computer:
 
-1.  Install the [Hugo](https://gohugo.io/getting-started/installing/) version specified in [`website/netlify.toml`](https://raw.githubusercontent.com/kubernetes/website/main/netlify.toml).
+1. Install the [Hugo](https://gohugo.io/getting-started/installing/) version specified in
+   [`website/netlify.toml`](https://raw.githubusercontent.com/kubernetes/website/main/netlify.toml).
```
Comment (Sea-n) [inline content/en/docs/contribute/new-content/open-a-pr.md:336]: I think it's not necessary to provide raw file link.
```suggestion
   [`website/netlify.toml`](https://git.k8s.io/website/netlify.toml).
```

**Comment by reylejano** [comment]
I agree with the reformat
/approve

**Review by Sea-n — COMMENTED** [review]
Those changes makes the document much better, looks good to me!
/lgtm

(My previous comment are all optional, and now I know it's unnecessary from comments in other PR.)

**Pull request merged**